tests/pass0/t03_select_bool.test

```diff
@@ -0,0 +1,22 @@
+discard """
+  output: "(Done 2)"
+"""
+(TypeDefs
+  (Int 8)
+  (ProcTy (Type 0))
+)
+(GlobalDefs)
+(ProcDefs
+  (ProcDef (Type 1) (Locals) (Continuations
+    (Continuation (Params) 0
+      (SelectBool (IntVal 1) (Continue 1) (Continue 2))
```

Comment:
Maybe I'm misreading/understanding the code, my impression was that it was `SelectBool <bool_expr> <goto_true> <goto_false>`, wouldn't `IntVal 1` result in a truthy value and therefore we'd go to continuation 1, which is `(IntVal 1)` and not `(IntVal 2)`?

---

N.B.: I haven't read the implementation yet, so the uncertainty around `SelectBool` below might be answered then.

Reply:
It's inverted (i.e., the first goto is for the 'false' case). I wasn't sure which way around is more intuitive, so I went with the one that maps directly to the `Branch x 0` instruction. There's no run-time cost to swapping the meaning, though.

Reply:
As is is fine, just takes a second, maybe a comment in the grammar at most.

Reply:
I've marked the gotos with which case the belong to in the grammar.